breguisvcmsg/00deps/linux/00libbreguisvcmsg.linux.depends

```diff
@@ -0,0 +1,1517 @@
+src/s_breguisvc_messages.linux.o: \
```

Comment:
Please remove this whole directory breguisvcmsg from repo as it is not required.